actions/createList.js

```diff
@@ -1,96 +1,65 @@
-import fs from 'fs';
-import path, { dirname } from 'path';
-import { fileURLToPath } from 'url';
+import fs from "fs";
+import path, { dirname } from "path";
 
-function getAllFiles(dirPath, arrayOfFiles) {
-  let files = fs.readdirSync(dirPath)
+const ROOT_PATH = path.resolve("./chains");
+const CHAIN_INFO_FILE = "chain-information.json";
+const CHAIN_LOGO_FILE = "chain-logo.png";
+const NATIVE_TOKEN_LOGO_FILE = "token-logo.png";
+const CONTRACT_TOKEN_INFO_FILE = "contract-information.json";
+const CHAINS_FOLDER_URL =
+  "https://github.com/ava-labs/subnet-assets/tree/main/chains";
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const TOKEN_LIST_FILE = path.resolve(__dirname, "../tokenList.json");
+import { fileURLToPath } from "url";
 
-  arrayOfFiles = arrayOfFiles || []
-
-  files.forEach(function (file) {
-    if (fs.statSync(dirPath + "/" + file).isDirectory()) {
-      arrayOfFiles = getAllFiles(dirPath + "/" + file, arrayOfFiles)
-    } else if (path.extname(file) == '.json') {
-      arrayOfFiles.push(path.join(dirPath, "/", file))
-    }
-  })
-
-  return arrayOfFiles
+function getTokens(chainId, chainTokenIds) {
```

Comment:
Similar to my other comment, `getTokens` isn't going to get the `logo.png` images stored for many tokens in this repo because we don't have `logoUrl` attributes for any of them since we were relying on the deterministic path of the image url.

See other comment for more details, but I think the best solution would be to add a `logoURI` entry to the `contract-information.json` for each token that has a `logo.png` in the token's directory.

Reply:
Yeah sorry about the back and forth on that @dovbrand .  I think including it is the right move though.